### DISCUSSION TITLE
labels with Select

### DISCUSSION THREAD

**Discussion by alexnguyennz** [melt-ui/melt-ui#194] [Q&A] [answered]
Hi is it possible to use labels with Select? Like on [this page](https://www.melt-ui.com/docs/builders/label) but with Select e.g. where clicking on labels gives focus to normal selects. Thanks.

**Comment by huntabyte** [comment]
Hey @alexnguyennz, sorry for the delayed response! It should certainly be possible! If not, please open an issue!

**Comment by alexnguyennz** [comment]
No worries, I gave it a try at the time but wasn't able to get it working.

**Reply by TGlide** [reply, marked as answer]
Here it is: https://github.com/melt-ui/melt-ui/issues/244

**Reply by alexnguyennz** [reply]
Awesome thanks!
